packages/dso-toolkit/reference/render/form--simple-form.html

```diff
@@ -14,7 +14,7 @@
         </label>
       </div>
       <div class="dso-selectable">
-        <input type="radio" id="Simpel-form-plat-dak-1" name="plat-dak" value="0">
```

Comment:
Waarom zijn hier de values ineens leeg? Als hij niet is ingevuld verwacht je hier toch een `value=""`?

Reply:
Goed gevonden, deze heb ik gemist. Er zijn namelijk ook situaties dat `value=""` verdwijnt, waar ik nog niet over uit ben of dat correct is. Maar `value="0"` moet gewoon blijven staan.